test/logic/QuotientTest.java

```diff
@@ -61,7 +61,7 @@ public void quotientConstructorShouldAddANewChannel() {
         assertEquals(quotient.getInputs().size(), 1);
         assertTrue(contains_channel_i_new);
     }
-
+    
```

Comment:
Remove white space characters so it becomes an empty line.

Reply:
Good catch.